Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -32,12 +32,13 @@ extension BrowserViewController {
     func destinationMenuSection(_ menuController: MenuViewController) -> some View {
         VStack(spacing: 0) {
             MenuItemButton(icon: #imageLiteral(resourceName: "menu_bookmarks").template, title: Strings.bookmarksMenuItem) { [unowned self, unowned menuController] in
-                let vc = BookmarksViewController(folder: Bookmarkv2.lastVisitedFolder(), isPrivateBrowsing: PrivateBrowsingManager.shared.isPrivateBrowsing)
+                let vc = BookmarksViewController(folder: self.bookmarkAPI.lastVisitedFolder(), bookmarkAPI: self.bookmarkAPI, isPrivateBrowsing: PrivateBrowsingManager.shared.isPrivateBrowsing)
```

Comment:
nit: `self` can be omitted in instances of `[unowned self]` (only `[weak self]` requires actually writing `self?`

Reply:
Fix at https://github.com/brave/brave-ios/pull/4024/commits/413ccf69542e430b625ce95596a97b57c8253388